MAINTAIN.md

```diff
@@ -55,6 +55,23 @@ has a major bug:
     - The [nightly job](https://github.com/neovim/bot-ci/blob/master/ci/nightly.sh)
       will update the release assets based on the `stable` tag.
 
+Third-party dependencies
+--------------
+
+The following are the third-party dependencies used in neovim.
+  - [Klib](https://github.com/attractivechaos/klib)
+  - [Lua](https://www.lua.org/download.html)
+  - [LuaJIT](https://github.com/LuaJIT/LuaJIT)
+  - [Luv](https://github.com/luvit/luv)
+  - [libmpack](https://github.com/libmpack/libmpack)
+  - [libtermkey](https://github.com/neovim/libtermkey)
+  - [libuv](https://github.com/libuv/libuv)
+  - [libvterm](http://www.leonerd.org.uk/code/libvterm/)
+  - [lua-compat](https://github.com/keplerproject/lua-compat-5.3)
+  - [tree-sitter](https://github.com/tree-sitter/tree-sitter)
+  - [xdiff](https://github.com/git/git/tree/master/xdiff)
+  - [lua-cjson](https://github.com/openresty/lua-cjson)
```

Comment:
there are 2 subcategories. 

1. The ones we need to manually update: lua-cjson, xdiff, mpack
2. The ones we need to "bump" in `third-party/CMakeLists.txt`

Reply:
almost, klib is inlined so I would list it here also. Though it doesn't live in src/, we kinda forked it.